src/components/App.js

```diff
@@ -12,27 +15,45 @@ export const styles = {
   },
 };
 
-function App({ classes }) {
-  return (
-    <React.Fragment>
-      <TopBar />
-      <Grid container>
-        <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
-          <CalendarViewContainer />
-        </Grid>
+export const mapDispatchToProps = dispatch => ({
+  fetchSchools: () => {
+    dispatch(fetchSchools('4720'));
+  },
+});
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.classes = props.classes;
+    this.fetchSchools = props.fetchSchools;
+  }
+
+  componentDidMount() {
+    this.fetchSchools();
+  }
 
-        <Grid item md={3} sm={12} xs={12}>
-          <SidebarView />
+  render() {
+    return (
+      <React.Fragment>
+        <TopBar />
+        <Grid container>
+          <Grid item md={9} sm={12} xs={12} className={this.classes.calendar}>
+            <CalendarViewContainer />
+          </Grid>
+          <Grid item md={3} sm={12} xs={12}>
+            <SidebarView />
+          </Grid>
         </Grid>
-      </Grid>
-    </React.Fragment>
-  );
+      </React.Fragment>
+    );
+  }
 }
 
 App.propTypes = {
   classes: PropTypes.objectOf(PropTypes.string).isRequired,
+  fetchSchools: PropTypes.func.isRequired,
 };
 
 export { App as UnstyledApp };
 
-export default withStyles(styles)(App);
+export default connect(null, mapDispatchToProps)(toJS(withStyles(styles)(App)));
```

Comment:
Just for clarity's sake, can you break this out into two lines?
`const AppWithStyles = withStyles(styles)(App);`
`export default connect(null, mapDispatchToProps)(AppWithStyles);`